source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
Thanks to @katebee I just learning that Python 3 is changing things so that `5/2` returns `2.5`! So interesting. I imagine we are teaching Python 2... what do you guys think @maxspencer @susiecoleman @gtrufitt